docs/source/api.rst

```diff
@@ -46,125 +167,87 @@ Random Circuit Functions
     retworkx.undirected_gnm_random_graph
     retworkx.random_geometric_graph
 
-Algorithm Functions
-===================
+.. _layout-functions:
 
-Specific Graph Type Methods
----------------------------
+Layout Functions
+================
+
+.. autosummary::
+   :toctree: stubs
+
+   retworkx.random_layout
+   retworkx.spring_layout
+   retworkx.graph_random_layout
+   retworkx.digraph_random_layout
```

Comment:
Did you want to move these to the separate per type api functions?

Reply:
Nice catch, I did overlook those two! They're separate now